engine/src/main/java/org/entando/entando/web/page/validator/PageValidator.java

```diff
@@ -109,4 +123,21 @@ public void validatePagesStatus(String pageCode, PagePositionRequest pageRequest
         }
     }
 
+    public void validateJsonPatchRequest(String pageCode, JsonNode jsonPatch, Errors errors) {
+        try {
+            jsonPatchValidator.validatePatch(jsonPatch);
+        } catch (InvalidJsonPatchException e) {
+            errors.reject(PageController.ERRCODE_INVALID_PATCH, new String[]{pageCode}, "page.update.patch.invalid");
+        }
+
+        for (JsonNode node : jsonPatch) {
+            String operationPath = node.get("path").asText();
+
+            if (operationPath.equals("/code")) {
+                errors.reject(PageController.ERRCODE_INVALID_PATCH, new String[]{pageCode}, "page.update.patch.invalid.code");
```

Comment:
Here you can pass an empty array to the reject method, because the pageCode is not used in the `page.update.patch.invalid.code` error message.